NeoML/test/src/TransformerSourceMaskTest.cpp

```diff
@@ -0,0 +1,68 @@
+/* Copyright © 2017-2022 ABBYY Production LLC
```

Comment:
This file isn't mentioned in corresponding CMakeLists.txt. These tests won't be compiled (and the launched)